server/belga/io/feed_parsers/belga_dpa_newsml_2_0.py

```diff
@@ -175,9 +175,15 @@ def parse_inline_content(self, tree, item):
     def parse_item_meta(self, tree, item):
         super().parse_item_meta(tree, item)
         meta = tree.find(self.qname("itemMeta"))
-        edNote = meta.find(self.qname("edNote"))
-        text = ElementTree.tostring(edNote, encoding="utf-8", method="text")
-        item["ednote"] = text.decode("utf-8").replace(" \n", "").replace("  ", "")
+        item["ednote"] = "\n".join(
+            line.strip()
+            for edNote in meta.findall(self.qname("edNote"))
+            if edNote.attrib.get("role") == "dpaednoterole:notepad"
+            for line in ElementTree.tostring(edNote, encoding="utf-8", method="text")
```

Comment:
I would still put this `ElementTree.tostring` in a helper function and use there some `try/catch` to avoid that issue in future